server/py_files/models/NeuralNet/NeuralNetClassifier.py

```diff
@@ -45,16 +47,11 @@ def train(self, samples, labels):
         self.model = model
 
         numeric_labels = SentenceLabelEncoder().encode_numerical(labels)
-
-        class_weights = class_weight.compute_class_weight('balanced',
-                                                 np.unique(numeric_labels),
-                                                 numeric_labels)
-
+        class_weights = class_weight.compute_class_weight('balanced', np.unique(numeric_labels), numeric_labels)
         y_train = SentenceLabelEncoder().encode_categorical(labels)
 
-        optimizer = adam(lr=0.00075)
```

Comment:
also have removed this - wasn't getting used
Cc @astrebeck

Reply:
okay